src/app/login/register/register.page.ts

```diff
@@ -44,8 +49,7 @@ export class RegisterPage implements OnInit {
       gender: new FormControl('', Validators.required),
       birthdate: new FormControl('', Validators.required),
       password: new FormControl('', Validators.compose([
-        Validators.minLength(6),
-        Validators.required
+        Validators.minLength(5)
```

Comment:
As per defect, password must be 8 character min.
Defect: https://slatein.atlassian.net/browse/ST-60

Reply:
Firebase only support minimum 6 character. As of now please consider for 6 character.

Reply:
Aprroved